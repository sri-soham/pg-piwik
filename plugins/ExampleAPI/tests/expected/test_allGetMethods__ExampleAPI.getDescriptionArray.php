a:5:{i:0;a:1:{i:0;s:5:"piwik";}i:1;a:1:{i:0;s:11:"open source";}i:2;a:1:{i:0;s:13:"web analytics";}i:3;a:1:{i:0;s:4:"free";}i:4;a:1:{i:0;s:45:"Strong message: Свободный Тибет";}}