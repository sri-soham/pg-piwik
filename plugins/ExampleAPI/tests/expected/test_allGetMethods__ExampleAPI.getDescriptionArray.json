[["piwik"],["open source"],["web analytics"],["free"],["Strong message: \u0421\u0432\u043e\u0431\u043e\u0434\u043d\u044b\u0439 \u0422\u0438\u0431\u0435\u0442"]]